base-theme/layouts/partials/footer.html

```diff
@@ -1,112 +1,114 @@
 <footer id="footer-container">
-  <div class="row pb-4 mx-0 justify-content-between">
-    <div>
-      <a
-        id="open-learning-logo"
-        href="https://openlearning.mit.edu/"
-        target="_blank"
+  <div id="home-footer" class="w-100 mx-auto">
```

Comment:
Ran `prettier` here because there was a lot of confusing inconsistent formatting.

Most of the code is same apart from a few differences such as image names, classes, etc.